style/golf.mss

```diff
@@ -0,0 +1,66 @@
+#landcover[zoom >= 14] {
+  ::high-zoom[zoom >= 12] {
+    [feature = 'golf_green'],
+    [feature = 'golf_tee'] {
+      polygon-fill: mix(@grass, @park, 50%);
```

Comment:
Makes sense. I have replaced the `mix()` with a `lighten()` on `@grass`, which I feel better represents the golf green. If you would prefer a specific color definition instead of a modification, I can do that too.